version1/04 - Web services.md

```diff
@@ -1150,10 +1152,18 @@ Content-Type: application/json
 ###### Details
 
 - **type**
-  - object type \[Saksmappe, Mappe, Moetemappe, AbstraktMappe, Journalpost, Basisregistrering, Moeteregistrering, Arkivnotat,
-    AbstraktRegistrering, Dokument\]
+  - object type \[Arkiv, Arkivskaper, Arkivdel, Klassifikasjonssystem, Klasse, Saksmappe, Mappe, Moetemappe, Moetedeltaker, AbstraktMappe, Journalpost, Basisregistrering, Moeteregistrering, Arkivnotat,
```

Comment:
Shouldn't we include `Sakspart` and `Korrespondansepart`?

Reply:
I think I've missed them, yes. I'll add a suggestion on the next line and commit it.

Reply:
Added with https://github.com/documaster/noark5-web-services/pull/70/commits/4d7fcc4d19830942c8f2878f4da0225376ee6f27